### PR TITLE
delete useless para groupversion.go

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
I think the para is  redundant,thanks!
**Release note**:

```release-note
```NONE

### CODE REVIEW

**Comment by jianglingxia** [comment]
ping @sttts
thanks very much for you  review！

File: staging/src/k8s.io/apiserver/pkg/endpoints/groupversion.go [hunk 1]
```diff
@@ -158,5 +158,3 @@ type staticLister struct {
 func (s staticLister) ListAPIResources() []metav1.APIResource {
 	return s.list
 }
-
-var _ discovery.APIResourceLister = &staticLister{}
```
Comment (caesarxuchao) [inline staging/src/k8s.io/apiserver/pkg/endpoints/groupversion.go]: Why do you remove the check?

Comment (sttts) [inline staging/src/k8s.io/apiserver/pkg/endpoints/groupversion.go]: @jianglingxia this is a type assertion in go, useful code.

**Pull request closed without merging**